automation/system_test/dev_utilities.py

```diff
@@ -0,0 +1,369 @@
+        },
+        "mysql": {
+            "chart": "bitnami/mysql",
+            "set_values": "--set primary.service.nodePorts.mysql=31003",
+        },
+    }
+    namespace = "devtools"
+    # Add Helm repos
+    add_repos()
+    # Check if the namespace exists, if not create it
+    check_namespace_cmd = f"kubectl get namespace {namespace}"
+    try:
+        subprocess.run(check_namespace_cmd.split(), check=True)
+    except subprocess.CalledProcessError:
+        create_namespace_cmd = f"kubectl create namespace {namespace}"
+        subprocess.run(create_namespace_cmd.split(), check=True)
+    for service, data in services.items():
+        if locals().get(service):
+            chart = data["chart"]
+            set_values = data["set_values"]
+            cmd = f"helm install {service} {chart} {set_values} --namespace {namespace}"
+            print(cmd)
+            subprocess.run(cmd.split(), check=True)
+    if redisinsight:
+        install_redisinsight(ipadd)
+
+
+@click.command()
+@click.option("--redis", is_flag=True, help="Uninstall Redis")
+@click.option("--kafka", is_flag=True, help="Uninstall Kafka")
+@click.option("--mysql", is_flag=True, help="Uninstall MySQL")
+@click.option("--redisinsight", is_flag=True, help="Uninstall Redis GUI")
+def uninstall(redis, kafka, mysql, redisinsight):
+    services = ["redis", "kafka", "mysql", "redisinsight"]
+    namespace = "devtools"
+    try:
+        if redisinsight:
+            cmd = "kubectl delete ingress -n devtools redisinsight"
+            subprocess.run(cmd.split(), check=True)
+        for service in services:
+            if locals().get(service):
+                cmd = f"helm uninstall {service} --namespace {namespace}"
+                subprocess.run(cmd.split(), check=True)
+        print("namespace deleteted")
+        delns = "kubectl delete namespace  devtools"
+        subprocess.run(cmd.split(), check=True)
+    except Exception as e:  # !!!
+        print(e)
+        pass
+        # code to handle any exception
+
+
+@click.command()
+def list_services():
+    namespace = "devtools"
+    # for service in services:
+    cmd = f"helm ls  --namespace {namespace} "
+    subprocess.run(cmd.split(), check=True)
+
+
+def list_services_h():
+    namespace = "devtools"
+    return get_installed_releases(namespace)
+
+
+@click.command()
+@click.option("--redis", is_flag=True, help="Install Redis")
+@click.option("--kafka", is_flag=True, help="Install Kafka")
+@click.option("--mysql", is_flag=True, help="Install MySQL")
+@click.option("--redisinsight", is_flag=True, help="Install Redis GUI")
+def status(redis, kafka, mysql, redisinsight):
+    namespace = "devtools"
+    if redis:
+        svc_password = get_svc_password(namespace, "redis", "redis-password")
+        print_svc_info(
+            "redis-master-0.redis-headless.devtools.svc.cluster.local",
+            6379,
+            "default",
+            svc_password,
+            "-------",
```

Comment:
printing password doesn't feel secure to me

Reply:
it only when u call function status, it for next steps